src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from "react";
-import WTRoutes from "./components/WTRoutes";
 
-const App: React.FC<{}> = () => {
-    return (
-        <WTRoutes />   
-    );
+// import Listings from "./components/Listings";
+import Filter from "./components/Filter"
+// import Signup from "./components/Signup";
+// import Login from "./components/Login"
+function App() {
+  return (
+    <>
+      <Filter />
+    </>
```

Comment:
revert this logic and keep the one in the main branch

Reply:
logic reverted. APP.js consistent with the 1.0 branch.